config/device.go

```diff
@@ -94,6 +100,17 @@ func ParseFromReader(defConfig *Device, rdr io.Reader) (*Device, error) {
 	// This may contain environment variables,
 	// so be kind and try to resolve
 	defConfig.General.NCSToolChainBase = resolveStringEnv(defConfig.General.NCSToolChainBase)
+
+	// If the toolchain base is not set, set it to the default value.
+	if defConfig.General.NCSToolChainBase == "" {
+		defConfig.General.NCSToolChainBase = func() string {
+			if runtime.GOOS == "windows" {
+				return "C:\\ncs"
+			}
+			return "~/ncs"
+		}()
+	}
+
```

Comment:
This can be moved [here](https://github.com/ffenix113/zigbee_home/pull/72/files#r1676865077). If by this point there is no value set - this is something that user should address, as default configuration will always have the path.